### PR TITLE
feat(anta.tests): Added the testcase to verify hostname, location and contact

### LINKED ISSUE #521: Add the test case to verify the system information(name, location, contact)

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
# Description

Added the testcase to verify hostname, location and contact details.

Fixes #521 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have run pre-commit for code linting and typing (`pre-commit run`)
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes (`tox -e testenv`)

### CODE REVIEW

File: anta/tests/snmp.py [hunk 1]
```diff
@@ -114,3 +114,63 @@ def test(self) -> None:
...
+    This class verifies the SNMP location of a device.
+
+    Expected results:
+        * success: The test will pass if the SNMP location matches the provided input.
+        * failure: The test will fail if the SNMP location does not match the provided input.
+    """
+
+    name = "VerifySnmpLocation"
+    description = "Verifies the SNMP location of a device."
+    categories = ["security"]
```
Comment (MaheshGSLAB) [inline anta/tests/snmp.py:130]: going to change the categories in next commit. missed to update. Same for  VerifySnmpContact

File: anta/tests/security.py [hunk 2]
```diff
@@ -377,3 +377,33 @@ def test(self) -> None:
...
+    This class verifies the hostname of a device.
+
+    Expected results:
+        * success: The test will pass if the hostname matches the provided input.
+        * failure: The test will fail if the hostname does not match the provided input.
+    """
+
+    name = "VerifyHostname"
+    description = "Verifies the hostname of a device."
+    categories = ["security"]
```
Comment (carl-baillargeon) [inline anta/tests/security.py:393]:
```suggestion
    categories = ["services"]
```

Comment (carl-baillargeon) [reply anta/tests/security.py:393]: Can you move this test to `services` please?

File: anta/tests/snmp.py (hunk 1 above)
Comment (carl-baillargeon) [reply anta/tests/snmp.py:130]: Yes please. There is an `snmp` category :)
```suggestion
    categories = ["snmp"]
```

File: anta/tests/snmp.py [hunk 3]
```diff
@@ -114,3 +114,63 @@ def test(self) -> None:
...
+    This class verifies the SNMP contact of a device.
+
+    Expected results:
+        * success: The test will pass if the SNMP contact matches the provided input.
+        * failure: The test will fail if the SNMP contact does not match the provided input.
+    """
+
+    name = "VerifySnmpContact"
+    description = "Verifies the SNMP contact of a device."
+    categories = ["security"]
```
Comment (carl-baillargeon) [inline anta/tests/snmp.py:160]:
```suggestion
    categories = ["snmp"]
```

**Review by carl-baillargeon — APPROVED** [review]
LGTM after changing the categories of the tests.

**Review by titom73 — APPROVED** [review]

**Pull request merged**